projects/1.16/assets/draconic-evolution/draconicevolution/lang/zh_cn.json

```
@@ -288,306 +297,311 @@
+  "item.draconicevolution.chaotic_proj_damage_module": "混沌箭矢伤害模块",
+  "item.draconicevolution.chaotic_proj_grav_comp_module": "混沌箭矢重力减少模块",
+  "item.draconicevolution.chaotic_proj_penetration_module": "混沌箭矢穿透模块",
+  "item.draconicevolution.chaotic_proj_velocity_module": "混沌箭矢速度模块",
   "item.draconicevolution.chaotic_shield_capacity_module": "混沌护盾容量模块",
   "item.draconicevolution.chaotic_shield_control_module": "混沌护盾控制模块",
-  "item.draconicevolution.chaotic_shield_recovery_module": "混沌护盾恢复模块",
-  "item.draconicevolution.chaotic_shovel": "混沌锹",
+  "item.draconicevolution.chaotic_shield_recovery_module": "混沌护盾回复模块",
+  "item.draconicevolution.chaotic_shovel": "混沌铲",
   "item.draconicevolution.chaotic_speed_module": "混沌速度模块",
-  "item.draconicevolution.chaotic_staff": "混沌之力权杖",
+  "item.draconicevolution.chaotic_staff": "混沌力量权杖",
   "item.draconicevolution.chaotic_sword": "混沌剑",
-  "item.draconicevolution.chaotic_undying_module": "混沌Undying Module",
-  "item.draconicevolution.creative_capacitor": "创造电容",
+  "item.draconicevolution.chaotic_undying_module": "混沌不死模块",
+  "item.draconicevolution.creative_capacitor": "创造通量容器",
   "item.draconicevolution.crystal_binder": "水晶绑定扳手",
   "item.draconicevolution.dislocator": "传送符咒",
   "item.draconicevolution.draconic_aoe_module": "神龙范围模块",
   "item.draconicevolution.draconic_auto_feed_module": "神龙自动进食模块",
   "item.draconicevolution.draconic_axe": "神龙斧",
   "item.draconicevolution.draconic_bow": "神龙弓",
-  "item.draconicevolution.draconic_capacitor": "神龙电容",
-  "item.draconicevolution.draconic_chestpiece": "神龙护胸",
+  "item.draconicevolution.draconic_capacitor": "神龙通量容器",
+  "item.draconicevolution.draconic_chestpiece": "神龙胸甲",
   "item.draconicevolution.draconic_damage_module": "神龙伤害模块",
-  "item.draconicevolution.draconic_energy_core": "神龙能量控制器",
+  "item.draconicevolution.draconic_energy_core": "神龙能量核心",
   "item.draconicevolution.draconic_energy_module": "神龙能量模块",
   "item.draconicevolution.draconic_flight_module": "神龙飞行模块",
   "item.draconicevolution.draconic_hoe": "神龙锄",
   "item.draconicevolution.draconic_jump_module": "神龙跳跃模块",
-  "item.draconicevolution.draconic_large_shield_capacity_module": "神龙大型护盾容量模块",
+  "item.draconicevolution.draconic_large_shield_capacity_module": "神龙大护盾容量模块",
   "item.draconicevolution.draconic_pickaxe": "神龙镐",
-  "item.draconicevolution.draconic_proj_accuracy_module": "神龙弹射物精度模块",
-  "item.draconicevolution.draconic_proj_anti_immune_module": "Projectile Immunity Cancellation Module",
-  "item.draconicevolution.draconic_proj_damage_module": "神龙弹射物伤害模块",
-  "item.draconicevolution.draconic_proj_grav_comp_module": "神龙弹射物重力补偿模块",
-  "item.draconicevolution.draconic_proj_penetration_module": "神龙弹射物破防模块",
-  "item.draconicevolution.draconic_proj_velocity_module": "神龙弹射物伤害模块",
+  "item.draconicevolution.draconic_proj_accuracy_module": "神龙箭矢精度模块",
+  "item.draconicevolution.draconic_proj_anti_immune_module": "神龙箭矢免疫远程取消模块",
```

好长......有没有简短点的
“反箭矢免疫”（怎么又是反），但是加上前缀又太重复了

---

```suggestion
  "item.draconicevolution.draconic_proj_anti_immune_module": "神龙箭矢反免疫模块",
```
但是会出现“伤害免疫”和“远程免疫”的歧义